meta-resin-common/recipes-core/systemd/systemd/0001-resolved-fix-loop-on-packets-with-pseudo-dns-types-229.patch

```diff
@@ -0,0 +1,45 @@
+From 9f939335a07085aa9a9663efd1dca06ef6405d62 Mon Sep 17 00:00:00 2001
```

Comment:
The missing Changelog-entry is on purpose?

Reply:
It defaults to the commit subject and since I find it descriptive enought, I left it as it is.